.github/workflows/helm-chart.yml

```diff
@@ -20,3 +20,34 @@ jobs:
         run: nix-shell --run ".ci/helm-docs.sh"
       - name: HelmChart Lint
         run: nix-shell --run ".ci/helm-lint.sh"
+
+  publish:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    needs: ["chart-test"]
+    if: ${{ github.event_name != 'pull_request' }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+          fetch-depth: 0
+    - uses: DeterminateSystems/nix-installer-action@v14
+    - uses: DeterminateSystems/magic-nix-cache-action@v8
+    - name: Pre-populate nix-shell
+      run: |
+        export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+        echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+        nix-shell --run "echo"
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Run chart-releaser
+      uses: helm/chart-releaser-action@v1.6.0
```

Comment:
Does this publish the release as well right?
How do we manage when to bump the chart version? On the other openebs repos we now do this via release branches, rather than develop, allowing for some more automated and manual testing before actually releasing.

Reply:
Yeah it will also create release

Reply:
Can you give me some example for that?

Reply:
We can mark versions as `pre` (with a suffix) when it's not running on a tag
Or when running on a branch other than `develop`

Reply:
The automation is here: https://github.com/openebs/zfs-localpv/blob/develop/.github/workflows/branch_preparation.yml
The chart publish here: https://github.com/openebs/zfs-localpv/blob/2a0e29a820e4c107e509ae2be9a03cdae3753b54/.github/workflows/release.yml#L210

Reply:
For that we have to refactor the entire CI pipelines that are related to build/publish to follow the branching that we want

Reply:
Alright, let's do this simple approach for now and introduce branching strategy later on.
But meanwhile, am I correct to assume as soon as this is merged a new release v0.9.0 would be done? And if so, do we want to do that now, or is there anything else we might want to do before releasing that? (nothing I can think of as of now, asking just in case)